### ISSUE TITLE
bug: yarn.lock file is empty

### ISSUE THREAD

**Issue by Parikshit2001** [code100x/daily-code#647]
The npm client installs dependencies into the node_modules directory non-deterministically. This means that based on the order dependencies are installed, the structure of a node_modules directory could be different from one person to another. These differences can cause works on my machine bugs that take a long time to hunt down.

[why to commit yarn.lock file](https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for)

**Comment by CuriousCoder00** [comment]
yeah, i can see the `yarn.lock` till the commit `ccf889b84861bd99b6d799c6474d58a328b534a1`
but from commit `3c99b48602b381fb227a8e88fd6d76065e211b5f` the `yarn.lock` is not there

**Comment by Parikshit2001** [comment]
@hkirat please assist how to fix this
